azurerm/internal/services/policy/virtual_machine_configuration_policy_assignments_resource.go

```diff
@@ -0,0 +1,259 @@
+package policy
```

Comment:
this file name is wrong: assingments vs assingment

Reply:
Oops, sorry I will fix this